paper/src/main/java/vg/civcraft/mc/namelayer/group/Group.java

```diff
@@ -319,6 +338,21 @@ public PlayerType getPlayerType(UUID uuid) {
 		}
 		return PlayerType.NOT_BLACKLISTED;
 	}
+
+	/**
+	 * @param uuid- The UUID of the player.
+	 * @return Returns the PlayerType of an invited UUID.
+	 */
+	public PlayerType getPlayerInviteType(UUID uuid) {
+		PlayerType member = invites.get(uuid);
+		if (member != null) {
+			return member;
+		}
+		if (NameLayerPlugin.getBlackList().isBlacklisted(this, uuid)) {
+			return null;
```

Comment:
Returning `null` invariably leads to NPEs down the line. Instead, use `java.util.Optional` or the `@Nullable`/`@NotNull` annotations.

Reply:
Do you think I should change this for `/nllm` as well? The only reason I have it the way it is is to try and stay consistent with `/nllm`

Reply:
I think not introducting new pitfalls matters the most. It'd be ideal if the codebase was consistent in its patterns around `null`, but that's probably more suited to a dedicated PR.